resolwe_bio/processes/variant_calling/ensembl_vep.py

```diff
@@ -14,6 +14,36 @@
 )
 
 
+def check_version(vep_output, stdout):
+    """Check version of Ensembl-VEP.
+
+    Example of the first part of an output from the command `vep --help`:
+    #----------------------------------#
+    # ENSEMBL VARIANT EFFECT PREDICTOR #
+    #----------------------------------#
+
+    Versions:
+    ensembl              : 104.1af1dce
+    ensembl-funcgen      : 104.59ae779
+    ensembl-io           : 104.1d3bb6e
+    ensembl-variation    : 104.6154f8b
+    ensembl-vep          : 104.3
+
+    Help: dev@ensembl.org , helpdesk@ensembl.org
+    Twitter: @ensembl
+    """
+    with open(vep_output, "w") as f:
+        f.write(stdout + "\n")
+    vep_version = int(
+        float(
+            [line for line in open(vep_output).readlines() if "ensembl-vep" in line][
+                0
+            ].split()[2]
```

Comment:
Writing to file might be redundant because it is not used anywhere else. Would something like `next((line for line in stdout.split("\n") if "ensembl-vep" in line)).split()[2]` be ok?